docs/data-sources/rds_available_flavors.md

```diff
@@ -0,0 +1,90 @@
+---
+subcategory: "Relational Database Service (RDS)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_rds_available_flavors"
+description: |-
+  Use this data source to get the specifications that a RDS instance can be changed to.
+---
+
+# huaweicloud_rds_available_flavors
+
+Use this data source to get the specifications that a RDS instance can be changed to.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+
+data "huaweicloud_rds_available_flavors" "test" {
+  instance_id           = var.instance_id
+  availability_zone_ids = "cn-north-4a"
+  ha_mode               = "ha"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the resource.
+  If omitted, the provider-level region will be used.
+
+* `instance_id` - (Required, String) Specifies the ID of the RDS instance.
+
+* `availability_zone_ids` - (Required, String) Specifies the availability zone.
```

Comment:
looks this argument can input more the one AZs, please update the description

Reply:
in reality, only one AZ is supported